src/components/admin/Tabs.tsx

```diff
@@ -4,13 +4,12 @@ import { useTenantDetails } from 'context/fetcher/Tenant';
 import { useMemo, useState } from 'react';
 import { useIntl } from 'react-intl';
 import { Link, useLocation } from 'react-router-dom';
-import { hasLength } from 'utils/misc-utils';
 
 function AdminTabs() {
     const intl = useIntl();
     const { pathname } = useLocation();
     const [selectedTab, setSelectedTab] = useState(0);
-    const tenantDetails = useTenantDetails();
+    const { hasTenants } = useTenantDetails();
```

Comment:
This is repeated... so threw in the length check into the context to reduce how often we were checking that.